### DISCUSSION TITLE
can ifcopenshell inspect the ifc class hierarchy?

### DISCUSSION THREAD

**Discussion by MichaelBelousov** [IfcOpenShell/IfcOpenShell#4834] [Q&A] [answered]
Can ifcopenshell inspect the class hierarchy? Ideally I'd like to do this in dynamically in python, but C++ or something else is ok.

**Comment by aothms** [comment, marked as answer]
See https://github.com/IfcOpenShell/IfcOpenShell/issues/953#issuecomment-677731329
